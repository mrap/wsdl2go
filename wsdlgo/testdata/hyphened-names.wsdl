<wsdl:definitions>
  <wsdl:types>
    <xsd:schema>
      <xsd:element name="my-type">
        <xsd:complexType>
          <xsd:all>
            <xsd:element name="my-field" type="string"/>
          </xsd:all>
        </xsd:complexType>
      </xsd:element>
    </xsd:schema>
  </wsdl:types>
</wsdl:definitions>
