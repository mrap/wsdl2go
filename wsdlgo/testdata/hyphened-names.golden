package internal

import ()

// MyType was auto-generated from WSDL.
type MyType struct {
	MyField string `xml:"my-field,omitempty" json:"my-field,omitempty" yaml:"my-field,omitempty"`
}
